src/plugins/console/public/application/components/overwrite_modal.test.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { shallowWithIntl } from 'test_utils/enzyme_helpers';
+import { OverwriteModal } from './overwrite_modal';
+import { act } from '@testing-library/react-hooks';
+import { ShallowWrapper } from 'enzyme';
+
+const confirmModalIdentifier = 'EuiConfirmModal';
+
+describe('OverwriteModal Component', () => {
+  let component: ShallowWrapper<any, Readonly<{}>, React.Component<{}, {}, any>>;
+  const mockOnConfirm = jest.fn();
+  const mockOnSkip = jest.fn();
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    act(() => {
+      component = shallowWithIntl(<OverwriteModal onConfirm={mockOnConfirm} onSkip={mockOnSkip} />);
+    });
+
+    component.update();
+  });
+
+  it('should render correclty', () => {
+    expect(component).toMatchSnapshot();
+  });
+
+  it('should call onConfirm when clicking the "Overwrite" button', () => {
+    act(() => {
+      // @ts-ignore
```

Comment:
nit: Why the ts-ignore?

Reply:
I saw a similar test in an existing plugin that used @ts-ignore. I tried various approaches but couldn't get it to work, so I added @ts-ignore. I'll keep trying to fix it.